Modules/Scripted/ScreenCapture/ScreenCapture.py

```diff
@@ -304,10 +312,20 @@ def setup(self):
     self.videoExportCheckBox.connect('toggled(bool)', self.videoFormatWidget, 'setEnabled(bool)')
     self.videoFormatWidget.connect("currentIndexChanged(int)", self.updateVideoFormat)
     self.singleStepButton.connect('toggled(bool)', self.numberOfStepsSliderWidget, 'setDisabled(bool)')
+    self.maxFramesWidget.connect('textChanged(QString)', self.maxFramesChanged)
 
     self.updateVideoFormat(0)
     self.updateViewOptions()
 
+  def maxFramesChanged(self):
```

Comment:
With QSpinBox you would then be sure to always have an `int` as parameter of the `valueChanged` signal